src/xSTIR/pSTIR/STIR.py

```diff
@@ -1505,42 +1516,34 @@ def get_linear_acquisition_model(self):
         check_status(am.handle)
         am.const = True # am to be a const reference of self
         return am
-
-    def direct(self, image, subset_num=0, num_subsets=1, out=None):
-        """Project an image into the (simulated) acquisition space.
-
-        Alias of forward.
-
-        Added for CCPi CIL compatibility
-        https://github.com/CCPPETMR/SIRF/pull/237#issuecomment-439894266
-        """
-        if not self.is_linear():
-            raise error('AcquisitionModel is not linear\nYou can get the ' +
-                        'linear part of the AcquisitionModel with ' +
-                        'get_linear_acquisition_model')
-        return self.forward(
-            image,
-            subset_num=subset_num,
-            num_subsets=num_subsets,
-            out=out)
-
-    def adjoint(self, ad, subset_num=0, num_subsets=1, out=None):
-        """Back-project acquisition data into image space.
-
-        Only if the AcquisitionModel is linear
-
-        Added for CCPi CIL compatibility
-        https://github.com/CCPPETMR/SIRF/pull/237#issuecomment-439894266
-        """
-        if not self.is_linear():
+    def direct(self, image, out = None):
```

Comment:
as I have learnt belatedly, it is considered good Python style not to have spaces around `=` when setting the argument default value (I know we have those spaces in many places, but let us at least not add more)